CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
```

Comment:
Hmm maybe we should explore a GEM file as an alternative if we have time.

Reply:
We can look into it after this, I guess?